lib/terrafying/aws.rb

```diff
@@ -56,6 +56,35 @@ def security_group(name)
           end
       end
 
+      def security_group_in_vpc(vpc_id, name)
+        @security_groups_in_vpc ||= {}
+        @security_groups_in_vpc[vpc_id + name] ||=
+          begin
+            STDERR.puts "Looking up id of security group '#{name}'"
```

Comment:
We can't put it on STDOUT ?

Reply:
Nope, this belongs on STDERR. It would screw up the `json` command if it was on STDOUT

Reply:
Oh, I see. OK so it looks good.